### ISSUE TITLE
[Bug]: Conflict with respawn anchor

### ISSUE THREAD

**Issue by DadoIrie** [someaddons/dusk#12]
### Describe the bug you're experiencing

with dusk when clicking on bed respawn anchor is set even tho respawn obelisk (https://www.curseforge.com/minecraft/mc-mods/respawn-obelisks) has removed it
because of that when clicking on the bed after having set the respawn anchor on the obelisk it gets re-set, but when dying player is not respawning on bed, but at world spawn where the player gets spawned when entering the first time the world after world creation (or server)

basically no respawn at bed nor obelisk

here's the log (at the end of that log I tried explaining my steps)
https://mclo.gs/0DfdPbG

### Reproducability

as explained above - also at the end of the log

### Mod up to date

- [X] Before submitting this issue I updated to the newest version and reproduced it

### Minecraft version

1.19.2

### Modloader version

Forge

### Logs

https://mclo.gs/0DfdPbG

**Comment by someaddons** [comment]
if respawn anchors wants to control all mods setting spawnpoints it probably should hook into the set spawnpoint call to create that, instead of hoping all mods avoid calling this

**Comment by DadoIrie** [comment]
what for is the spawnpoint used in dusk? 
I've communicated with respawn obelisk's dev and couldn't do much, because of dusk's closed source code.
Posted this incompatibility issue here in hopes it could get resolved somehow.

And `Player#setRespawnPosition` is being used to set the position and, when dusk is installed & enabled, the bed overwrites it without actually setting a respawn anchor which results to the deletion of players set respawn and since there is no more player set respawn anchor upon death the world's anchor will be used.

UPDATE: feel free closing this issue as I've found other means increasing difficulty on the server - left it open since it's not a resolved issue

**Comment by RedPxnda** [comment]
> if respawn anchors wants to control all mods setting spawnpoints it probably should hook into the set spawnpoint call to create that, instead of hoping all mods avoid calling this

I don’t want to control all mods setting spawn points. I just mixin to stop beds from doing so.

**Comment by someaddons** [comment]
> what for is the spawnpoint used in dusk? I've communicated with respawn obelisk's dev and couldn't do much, because of dusk's closed source code. Posted this incompatibility issue here in hopes it could get resolved somehow.
> 
> And `Player#setRespawnPosition` is being used to set the position and, when dusk is installed & enabled, the bed overwrites it without actually setting a respawn anchor which results to the deletion of players set respawn and since there is no more player set respawn anchor upon death the world's anchor will be used.
> 
> UPDATE: feel free closing this issue as I've found other means increasing difficulty on the server - left it open since it's not a resolved issue

dusk is visible source not closed source, so that shouldnt be an issue

**Comment by someaddons** [comment]
fixed in 3.4
